atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -340,7 +348,7 @@ def run(self):
             session = self.get_registry_session(registry)
 
             if self.group:
-                digest = self.group_manifests_and_tag(session, source)
+                return self.group_manifests_and_tag(session, source)
```

Comment:
This is within a loop.

Don't we want to append to a list here, and return the list when the loop finishes? (This looks to be a bug that slipped through in https://github.com/projectatomic/atomic-reactor/pull/846)

Reply:
Ah, yes, gonna fix it.

Its a bit weird though - we always use just first result from it anyway

Reply:
I'd also be happy with a check that there is only one item to iterate over, and a NotImplemented exception raised if there are more. :)

Reply:
This seems tricky - and also arguable.

We're preparing a manifest list using same worker builds - we're just pushing those to different registries, so essentially there is no difference which manifest list should be used later on.

Raising NotImplementedError here causes a lot of test failures - lets implement this in a separate story.